docs/accelerators.md

```diff
@@ -1,5 +1,16 @@
 # Using AI Accelerators {#ovms_docs_target_devices}
 
+## Prepare test model
+
+Download ResNet50 model
+
+```bash
+mkdir models
+docker run -u $(id -u):$(id -g) -v ${PWD}/models:/models openvino/ubuntu18_dev:latest omz_downloader --name resnet-50-tf --output_dir /models
```

Comment:
ubuntu18 OV images might be deprecated in near future, maybe use ubuntu20_dev?

Reply:
Corrected for all occurrences